website/docs/plugins/ax_sweeper.md

```diff
@@ -0,0 +1,55 @@
+---
+id: ax_sweeper
+title: Ax Sweeper plugin
+sidebar_label: Ax Sweeper plugin
+---
+This plugin provides a mechanism for Hydra applications to use the <a class="external" href="https://ax.dev/" target="_blank">Adaptive Experimentation Platform, aka Ax</a>. Ax can optimize any experiment - machine learning experiments, A/B tests, and simulations.
+
+Install with 
+```
+pip install hydra_ax_sweeper
+```
+
+Once installed, override `hydra/sweeper` in your config:
+
+```yaml
+defaults:
+  - hydra/sweeper: ax
+```
+
+and add a config corresponding to all the parameters that you want to optimize. For instance, the configuration corresponding to a parameter `x` may look like this:
+
+```
```

Comment:
indicate that the type configuration is optional.

Reply:
`type` is not optional when specified via the  config file. I am adding more information that how these values can be set via command line.

Reply:
Done